frontend/src/service/MonsterService.js

```diff
@@ -20,3 +20,13 @@ export const updateMonster = (id, userId, name, image, token) =>
   axios
     .put('/api/monster/' + id, { id, userId, name, image }, header(token))
     .then((response) => response.data);
+
+export const removeMonster = (id, userId, token) =>
```

Comment:
why did you change the writing style of this axios method (compared to get/post/put - methods)?

Reply:
axios.delete is not designed to send data. I have therefore decided to use this syntax.